src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/StateFlowEngine.java

```diff
@@ -368,28 +368,14 @@ public StateFlow evaluate(CaseDetails caseDetails) {
 
     public StateFlow evaluate(CaseData caseData) {
         if (caseData.getSuperClaimType() != null && caseData.getSuperClaimType().equals(SPEC_CLAIM)) {
-            return buildSpec().evaluate(caseData);
+            return build(SPEC_DRAFT).evaluate(caseData);
         }
-        return build().evaluate(caseData);
-    }
-
-    public StateFlow evaluateSpec(CaseDetails caseDetails) {
-        return evaluateSpec(caseDetailsConverter.toCaseData(caseDetails));
-    }
-
-    public StateFlow evaluateSpec(CaseData caseData) {
-        return buildSpec().evaluate(caseData);
+        return build(DRAFT).evaluate(caseData);
     }
 
     public boolean hasTransitionedTo(CaseDetails caseDetails, FlowState.Main state) {
         return evaluate(caseDetails).getStateHistory().stream()
             .map(State::getName)
             .anyMatch(name -> name.equals(state.fullName()));
     }
-
-    public boolean hasSpecTransitionedTo(CaseDetails caseDetails, FlowState.Main state) {
```

Comment:
Are the test classes also checked for usage of this method?

Reply:
yes, I just remove one test which was related to this method